spec/requests/event_spec.rb

```diff
@@ -0,0 +1,146 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Event endpoint', type: :request do
+  let!(:admin) { create(:admin) }
+  let!(:user) { create(:user) }
+  let!(:event) { create(:event) }
+  let!(:event_to_update) { create(:event) }
+  let!(:auth_headers) { admin.create_new_auth_token }
+  let!(:auth_headers_user) { user.create_new_auth_token }
+  let!(:invitation) { create(:invitation, user: user, event: event) }
+
+  let!(:event_to_create_with_user) do
+    {
+      'event' =>
+        {
+          'name' => 'evento_testing',
+          'address' => 'testing_address333',
+          'cost' => 200,
+          'start_time' => '2020-10-07T13:28:06.419Z',
+          'end_time' => '2020-10-07T15:28:06.419Z',
+          'cancelled' => false
+        },
+      'user_ids' => [user.id.to_i]
+    }
+  end
+  let!(:event_to_create_without_user) do
```

Comment:
Dejar un espacio entre cada let, y entre cada it, y entre cada describe, y entre cada context.